internal/cli/cli.go

```diff
@@ -137,6 +137,11 @@ func printError(io *iostreams.IOStreams, cs *iostreams.ColorScheme, cmd *cobra.C
 		fmt.Fprintln(io.ErrOut)
 	}
 
+	if requestId := flyerr.GetErrorRequestID(err); requestId != "" {
+		fmt.Fprintln(io.ErrOut, "Fly-Request-Id:", requestId)
```

Comment:
My thinking was that the request-id is a separate concern to the user than the actual error itself, though I'm down to change it. Would you want to give an example of how you think we should print it?

Reply:
```
Error: failed to update VM 148ed510a01458: request returned non-2xx status, 504 (request ID a9bf7c15f6b987cd-lga)
```

So that users can just share the one line with us when debugging. I don't think we need it for errors we handle already, such as "no capacity in ord", as it'll just make the output messier, and our error messages should have enough info for the users to debug.

Reply:
Looks good, will do!